portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageProfile.testcase

```diff
@@ -85,10 +85,38 @@ definition {
 		task ("Assert Page Profile") {
 			ACUtils.assertPageTitle(pageTitle = "AC Page - Site Name - ${dataSourceName}");
 
-			for (var cardTitle : list "Visitors Behavior,Audience,Views by Location,Views by Technology,Displayed Assets") {
+			for (var cardTitle : list "Visitors Behavior,Audience,Views by Location,Views by Technology") {
 				ACCards.viewCards(cardTitle = "${cardTitle}");
 			}
 		}
+
+		task ("View the time filter of Visitors Behavior is Last 24 Hours") {
+			ACTimeFilter.assertCurrentTimeFilterRange(
+				cardName = "Visitors Behavior",
+				timeFilterRange = "Last 24 hours");
+		}
+
+		for (var tabName : list "Path,Known Individuals") {
+			task ("Switch the tab") {
+				ACNavigation.switchTab(tabName = "${tabName}");
+			}
+
+			task ("Set time filter use 30 days") {
+				ACTimeFilter.clickTimeFilterButton();
+
+				ACTimeFilter.setLast30Days();
+			}
```

Comment:
Hey @ViniLopes20 just a small suggestion, do you think it would be better to use this `Set the time filter to the last 30 days`?